great_expectations/checkpoint/checkpoint.py

```diff
@@ -205,6 +227,7 @@ def run(
         result_format=None,
         **kwargs,
     ) -> List[ValidationOperatorResult]:
+        run_time = run_time or datetime.now()
```

Comment:
Note: The code in `_get_substituted_validation_dict` changed -- I do not know why github is not telling us to pull the latest from `checkpoint-class`.

Reply:
Oh. Do you remember the way it changed that affects the run_time defined here?